### ISSUE TITLE
MAC M1 issue to install DataSAIL

### ISSUE THREAD

**Issue by EasternCaveMan** [kalininalab/DataSAIL#12]
Hi Roman,
I did following step to install DataSAIL on mac m1,
1-  conda create -n sail python=3.10
2- conda activate sail
3- conda install -c conda-forge mamba
4-  mamba install -c kalininalab -c conda-forge -c mosek -c bioconda datasail
then I got this error:

```CLI
(sail) vahidatabaigi@vahids-MacBook-Pro ~ % mamba install -c kalininalab -c conda-forge -c bioconda datasail

Looking for: ['datasail']

kalininalab/osx-arm64 (check zst)                   Checked  0.3s
kalininalab/noarch (check zst)                      Checked  0.2s
bioconda/osx-arm64 (check zst)                      Checked  0.2s
bioconda/noarch (check zst)                         Checked  0.2s
pkgs/main/osx-arm64 (check zst)                    Checked  0.1s
pkgs/main/noarch (check zst)                       Checked  0.0s
pkgs/r/osx-arm64 (check zst)                       Checked  0.0s
pkgs/r/noarch (check zst)                          Checked  0.0s
bioconda/osx-arm64                                 124.0 B @   1.3kB/s  0.1s
kalininalab/osx-arm64                                1.4kB @   3.9kB/s  0.4s
kalininalab/noarch                                 116.0 B @ 227.0 B/s  0.5s
pkgs/r/osx-arm64                                   108.0 B @ 102.0 B/s  0.5s
pkgs/main/noarch                                   702.3kB @ 182.0kB/s  3.5s
pkgs/r/noarch                                        2.1MB @ 183.6kB/s 11.1s
pkgs/main/osx-arm64                                  2.1MB @ 152.1kB/s 12.6s
bioconda/noarch                                      5.1MB @ 230.8kB/s 18.3s
conda-forge/osx-arm64                                8.3MB @ 279.6kB/s 29.6s
conda-forge/noarch                                  13.3MB @ 376.9kB/s 35.3s

Pinned packages:
  - python 3.10.*


Could not solve for environment specs
The following package could not be installed
└─ datasail is not installable because it requires
   └─ cd-hit, which does not exist (perhaps a missing channel).

```
then I tried to install cd-hit directly from bioconda/conda-forge channels in none of them was available for mac m1. I alos tried with pip, wasnt available
```CLL
(sail) vahidatabaigi@vahids-MacBook-Pro ~ % conda install bioconda::cd-hit
Channels:
 - bioconda
 - defaults
 - conda-forge
Platform: osx-arm64
Collecting package metadata (repodata.json): done
Solving environment: failed

PackagesNotFoundError: The following packages are not available from current channels:

  - bioconda::cd-hit

Current channels:

  - https://conda.anaconda.org/bioconda
  - defaults
  - https://conda.anaconda.org/conda-forge
  - https://conda.anaconda.org/bioconda

To search for alternate channels that may provide the conda package you're
looking for, navigate to

    https://anaconda.org

and use the search bar at the top of the page.


(sail) vahidatabaigi@vahids-MacBook-Pro ~ % pip install cd-hit
ERROR: Could not find a version that satisfies the requirement cd-hit (from versions: none)
ERROR: No matching distribution found for cd-hit
```
However, I have installed cd-hit through brew as recomended in cd-hit github
https://github.com/weizhongli/cdhit
However it seems that datasil still can not recognised 

```CLI
(sail) vahidatabaigi@vahids-MacBook-Pro ~ % brew install cd-hit
==> Downloading https://formulae.brew.sh/api/formula.jws.json
##################################################################################################################################################################################################### 100.0%
==> Downloading https://formulae.brew.sh/api/cask.jws.json
##################################################################################################################################################################################################### 100.0%
Warning: brewsci/bio/cd-hit 4.8.1_1 is already installed and up-to-date.
To reinstall 4.8.1_1, run:
  brew reinstall cd-hit
(sail) vahidatabaigi@vahids-MacBook-Pro ~ % mamba install -c kalininalab -c conda-forge -c bioconda datasail

Looking for: ['datasail']

conda-forge/osx-arm64                                       Using cache
conda-forge/noarch                                          Using cache
bioconda/noarch                                             Using cache
pkgs/main/noarch                                              No change
pkgs/main/osx-arm64                                           No change
pkgs/r/noarch                                                 No change
bioconda/osx-arm64                                            No change
pkgs/r/osx-arm64                                              No change
kalininalab/osx-arm64                                         No change
kalininalab/noarch                                            No change

Pinned packages:
  - python 3.10.*


Could not solve for environment specs
The following package could not be installed
└─ datasail is not installable because it requires
   └─ cd-hit, which does not exist (perhaps a missing channel).
```
I am sure that I installed cd-hit successfully, becuase i did clustering with it. 
Best,
Vahid

**Comment by Old-Shatterhand** [comment]
Hi @atabaigi 
the problem is that CD-Hit is not available for M1 chips, i.e. the osx-arm64 architecture. I'll check out ways to fix it and use other installations of CD-Hit for M1.

**Comment by Old-Shatterhand** [comment]
This is a duplicate to #16. A fix will be provided in version 1.0.0. More details in #16.
